services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/HankeControllerTest.kt

```diff
@@ -59,6 +59,36 @@ class HankeControllerTest {
         Assertions.assertThat((response.body as Hanke).nimi).isNotEmpty() // Some compilation/build-setups apparently do not allow this .isNotEmpty without ()
     }
 
+    @Test
+    fun `test that the getAllHankeItems returns ok and two items`() {
+
+        var listOfHanke =  mutableListOf(
+            Hanke(1234, mockedHankeTunnus, true,
+                    "Mannerheimintien remontti remonttinen", "Lorem ipsum dolor sit amet...",
+                    getDatetimeAlku(), getDatetimeLoppu(), Vaihe.OHJELMOINTI, null,
+                    1, "Risto", getCurrentTimeUTC(), null, null, SaveType.DRAFT),
+            Hanke(50, "HAME50", true,
+                    "Hämeenlinnanväylän uudistus", "Lorem ipsum dolor sit amet...",
+                    getDatetimeAlku(), getDatetimeLoppu(), Vaihe.SUUNNITTELU, SuunnitteluVaihe.KATUSUUNNITTELU_TAI_ALUEVARAUS,
+                    1, "Paavo", getCurrentTimeUTC(), null, null, SaveType.SUBMIT)
+        )
+
+        Mockito.`when`(hankeService.loadListOfHanke()).thenReturn(listOfHanke)
+
+        val response: ResponseEntity<Any> = hankeController.getAllHankeItems()
+
+        //basic checks for getting a response
+        Assertions.assertThat(response.statusCode).isEqualTo(HttpStatus.OK)
```

Comment:
How about getting rid of "Assertions.assertThat" here and import it instead and use just "assertThat"?

Reply:
Ah.. I again forgot this clearer way.. :)